lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -658,6 +660,8 @@ SILPassPipelinePlan::getOnonePassPipeline(const SILOptions &Options) {
   P.startPipeline("Serialization");
   P.addSerializeSILPass();
 
+  P.addMandatoryCombiner();
```

Comment:
I think we want to add this before serialization.

Reply:
Fixed.